django_tables2/templatetags/django_tables2.py

```diff
@@ -198,6 +198,29 @@ class Meta:
     return RenderTableNode(table, template)
 
 
+
+RE_UPPERCASE = re.compile('[A-Z]')
+
+@register.filter
+@stringfilter
+def table_page_range(num_pages,arg):
```

Comment:
please add a docstring describing what this filter does. Also: always use a space after a comma. (I just see a commit coming in fixing this)